Formula/ponyc.rb

```diff
@@ -19,6 +19,10 @@ class Ponyc < Formula
   def install
     ENV.cxx11
 
+    on_linux do
+      inreplace "CMakeLists.txt", "PONY_COMPILER=\"${CMAKE_C_COMPILER}\"", "PONY_COMPILER=\"/usr/bin/gcc\""
```

Comment:
What happens if try to use `ENV.cc` instead of `/usr/bin/gcc`

Reply:
On my Ubuntu box, it builds and passes `test` and `audit`, so let's see if it works on CI.

(As a test, I tried removing it entirely, and it fails `audit` as described in https://github.com/Homebrew/linuxbrew-core/pull/20412.)